code/modules/mob/living/simple_animal/friendly/drone/verbs.dm

```diff
@@ -38,3 +38,31 @@
 	if(alert_s && A && stat != DEAD)
 		var/msg = "<span class='boldnotice'>DRONE PING: [name]: [alert_s] priority alert in [A.name]!</span>"
 		alert_drones(msg)
+
+/mob/living/simple_animal/drone/verb/cmd_robot_alerts()
+	set category = "Drone"
+	set name = "Show Alerts"
+	if(usr.stat == DEAD)
+		to_chat(src, "<span class='userdanger'>Alert: You are dead.</span>")
+		return //won't work if dead
+	robot_alerts()
+	
+/mob/living/simple_animal/drone/proc/robot_alerts()
+	var/dat = ""
+	for (var/cat in alarms)
+		dat += text("<B>[cat]</B><BR>\n")
+		var/list/L = alarms[cat]
+		if (L.len)
+			for (var/alarm in L)
+				var/list/alm = L[alarm]
+				var/area/A = alm[1]
+				dat += "<NOBR>"
```

Comment:
This is a deprecated non standard element. Replace it 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr

Reply:
I'll give it a try, but this is basically from the silicon code here. I have no fucking clue how any of this works, only that it does work, so any fix I do attempt to make might fuck it up even more, just an FYI. That being said, I don't have coder socks just these normal socks